nipype/algorithms/confounds.py

```diff
@@ -464,7 +471,7 @@ def _run_interface(self, runtime):
                              .format(self.inputs.realigned_file, imgseries.ndim, imgseries.shape))
 
         if isdefined(self.inputs.mask_file):
-            in_mask_data = nb.load(self.inputs.mask_file, mmap=NUMPY_MMAP).get_data()
+            in_mask_data = nb.load(self.inputs.mask_file[0], mmap=NUMPY_MMAP).get_data()
```

Comment:
@satra maybe you can take a look, there is a failing test I have to figure out first